package.json

```diff
@@ -8,7 +8,7 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-redux": "^7.2.5",
```

Comment:
- [ ] It would be better if you add `"react-redux"` to package.json. It will be needed to use Redux in the next projects. kindly add this. 
👍 💯